qsimcirq/qsim_simulator.py

```diff
@@ -382,3 +382,124 @@ def simulate_sweep(
       trials_results.append(result)
 
     return trials_results
+
+  def simulate_expectation_values_sweep(
```

Comment:
Discussed offline. This requires optimizations for noisy circuit simulations. This can be addressed later.

Reply:
If we need a cheap fix (i.e. without Cirq changes), we can memoize the translated circuit in QSimCircuit and instruct users to pass QSimCircuits to their simulators for less overhead. However, this only fixes one type of overhead and doesn't address the more general problem of representing noisy simulation in Cirq.

Opened #293 to track this.

Reply:
I think we can wait until the more general problem is addressed.